packages/server/graphql/mutations/addTeamMemberIntegrationAuth.ts

```diff
@@ -85,8 +85,8 @@ const addTeamMemberIntegrationAuth = {
       }
     } else if (scope === 'org' && teamId !== integrationProvider.teamId) {
       const [providerTeam, authTeam] = await Promise.all([
-        dataLoader.get('teams').load(integrationProvider.teamId),
-        dataLoader.get('teams').load(teamId)
+        dataLoader.get('teams').loadNonNull(integrationProvider.teamId),
```

Comment:
kudos to @Dschoordsch for this nice method. now it's catching all the possible nulls!